Makefile

```diff
@@ -373,3 +373,13 @@ release-push-%:
 	$(MAKE) clean-bin
 	$(MAKE) ARCH=$* build
 	$(MAKE) ARCH=$* push
+
+# SvCat Kubectl plugin stuff
+############################
+svcat: .init .generate_files \
+		$(BINDIR)/svcat/svcat
+
+$(BINDIR)/svcat/svcat: \
+		cmd/svcat/main.go
+	rm -rf $(BINDIR)/svcat
+	$(DOCKER_CMD) $(GO_BUILD) -o $@ $<
```

Comment:
What is `$<`?

Reply:
The name of the first prerequisite, in this case "cmd/svcat/main.go" form line 383. See [here.](http://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html#Automatic-Variables) I should probably change it to be consistent stylistically with the rest of the file, though.

Reply:
Neat! I did not know that. That seems really useful. Maybe we should use it in other places?